client/client.go

```diff
@@ -239,6 +246,136 @@ func (c *Client) Do(ctx context.Context, req *http.Request, v interface{}) error
 	return err
 }
 
+func searchSlice(slice []string, key string) bool {
+	for _, v := range slice {
+		if v == key {
+			return true
+		}
+	}
+	return false
+}
+
+// DoWithFilters performs request passed and filters entities in json response
+func (c *Client) DoWithFilters(ctx context.Context, req *http.Request, v interface{}, filters []*AdditionalFilter, baseSearchPaths []string) error {
+	req = req.WithContext(ctx)
+	resp, err := c.client.Do(req)
+
+	if err != nil {
+		return err
+	}
+
+	defer func() {
```

Comment:
Indentation of the code and spacings are not consistent with rest of the code

Reply:
I've run `gofmt` to iron out indentation issues. Is the spacing inconsistent due to unnecessary newlines? I've removed a bunch of newlines just now.